cmake/developer_package/compile_flags/sanitizer.cmake

```diff
@@ -17,12 +17,20 @@ if (ENABLE_SANITIZER)
             "https://github.com/openvinotoolkit/openvino/wiki/AddressSanitizer-and-LeakSanitizer")
         endif()
     elseif(CMAKE_COMPILER_IS_GNUCXX OR OV_COMPILER_IS_CLANG)
-        set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -fsanitize=address")
+        set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -fsanitize=address -fsanitize-blacklist=${OpenVINO_SOURCE_DIR}/tests/asan/ignore.txt")
+        if(BUILD_SHARED_LIBS)
+            set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -shared-libasan")
+        endif()
+
```

Comment:
@vurusovs 
Not sure whether it was intentional, but this makes ENABLE_SANITIZIER option to be working only with clang, since gcc has no -fsanitize-blacklist option